guarddog/analyzer/sourcecode/download-executable.yml

```diff
@@ -62,12 +62,6 @@ rules:
                             pattern-regex: (?i)^['"].*?\.exe['"]$
 
     pattern-sources:
-      - pattern: |
-          (...).send(...)
-          ...
-      - pattern: |
-          send(...)
-          ...
       - pattern: |
           (...).request(...)
           ...
```

Comment:
I would keep this one, is basically a variation of request.get depending on the import.
Actually it could be rewritten using `symbolic_propagation` like but I won't harm to keep it.
probably all requests matches the same as `requests.$FUNC(...)` with that option enabled

Reply:
sounds good